cmd/kinflate/README.md

```diff
@@ -1,12 +1,8 @@
-# kinflate
-
-`kinflate` is a kubernetes cluster configuration
-utility.
-
-It reads a cluster app definition from a
-directory of YAML files, and creates new YAML suitable
-for directly piping to `kubectl apply`.
+# Overview
 
+`kinflate` is a command line interface to manage Kubernetes Application 
+configuration.  It reads an app definition from a directory of YAML files, and 
```

Comment:
What is an  "app definition"? Is it Deployment/ReplicaSet/...? Does this not work with services?

Change YAML files to "YAML manifest files".